crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -108,28 +108,62 @@ pub(crate) enum EnumVariantFields<'a> {
     Unit,
 }
 
+#[derive(Clone, Copy, PartialEq, Eq)]
```

Comment:
Nit: Does this need to be `Copy` + `Clone`?
```suggestion
#[derive(PartialEq, Eq)]
```

Reply:
It doesn't, just a habit. I figured since it's trivially copyable then it would make things easier for someone else who wanted to use it down the line.

Reply:
I removed the extra derives. They can be added back in the future if needed.